markdown/events/2023/hackathon-march-2023/usa-university-texas.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Hackathon - March 2023 (Dallas)
+subtitle: Local node of the nf-core hackathon at the University of Texas at Dallas.
+type: hackathon
+start_date: '2023-03-27'
+start_time: '10:00 CEST'
```

Comment:
Same timezone  or the local one? Or will it translte automatically

Reply:
Should translate automatically

Reply:
should be a local one

Reply:
Yeah but it will render as whatever 10CEST is for the viewer, right? So it'll be stupid in the morning, but it's up to the host to update this page. I'm not going to tell them when they should start.